crates/ironrdp/examples/server.rs

```diff
@@ -106,10 +133,23 @@ fn setup_logging() -> anyhow::Result<()> {
     Ok(())
 }
 
-fn acceptor(cert_path: &str, key_path: &str) -> anyhow::Result<TlsAcceptor> {
+fn acceptor(cert_path: &str, key_path: &str) -> anyhow::Result<(TlsAcceptor, Vec<u8>)> {
```

Comment:
nitpick: Maybe use a struct to name the values. `Vec<u8>` on its own is not telling much.

```rust
struct AcceptResult {
    tls_acceptor: TlsAcceptor,
    public_key: Vec<u8>,
}

fn acceptor(cert_path: &str, key_path: &str) -> anyhow::Result<AcceptResult> { … }
```

or

```rust
struct PublicKey(Vec<u8>);

fn acceptor(cert_path: &str, key_path: &str) -> anyhow::Result<(TlsAcceptor, PublicKey)> { … }
```

But it’s not too much of a big deal since it’s an example. Feel free to address or not.